src-web/containers/App.js

```diff
@@ -170,6 +170,17 @@ class App extends React.Component {
               />
             )}
           />
+          <Route
+            exact
+            path={`${BASE_PAGE_PATH}/argo/appset`}
```

Comment:
@chenz4027 I've been thinking whether we should use `${BASE_PAGE_PATH}/argo/appset` or `${BASE_PAGE_PATH}/argoappset` because we have the URL pattern `${BASE_PAGE_PATH}/:namespace/:name` for accessing apps. If a user deploy an app called `appset` to the namespace `argo` they might not be able to access their app. Would be something to test when you have a chance :)

Reply:
True, I think in current code if the user deploys an app called appset to the namespace argo, it will be redirected to create argo appset page instead of viewing topology.

Reply:
`argoappset` is fine with me.